apps/wallet/src/features/Widget/BuiltInNetwork/CollectiblesTab.tsx

```diff
@@ -18,9 +18,16 @@ export const CollectiblesTab: FC<Props> = ({ collections = [] }) => {
 
 	const handlePressItem = (ele: WrappedCollection) => {
 		const collectionId = ele._id.split('/')[2];
+
 		navigate('Dashboard', {
 			screen: 'Explore',
-			params: { screen: 'Collection', params: { id: collectionId } },
+			params: {
+				screen: 'Collection',
+				params: {
+					screen: 'Default',
```

Comment:
can you explain the `Default` screen

Reply:
i think we'll have a collection stacks which contains the collection screen (default screen) and collectible screen, so that is the way i treat it a default screen